### PR TITLE
Add release notes for 1.10.1

### PR DESCRIPTION
**Pull request by Monkeyanator**
Relnotes for 1.10.1, just two made it into this one

### CODE REVIEW

File: content/en/news/releases/1.10.x/announcing-1.10.1/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.10.0 and Istio 1.10.1.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue causing the `Host` header to not be modifiable for specific destinations in `VirtualService` ([Issue #33226](https://github.com/istio/istio/issues/33226))
+
+- **Fixed** an issue making it impossible to set PDB `maxUnavailable` field in 'IstioOperator' ([Issue #31910](https://github.com/istio/istio/issues/31910))
```
Comment (ericvn) [inline content/en/news/releases/1.10.x/announcing-1.10.1/index.md:20]: Use backticks around IstioOperator

**Review by ZhiHanZ — APPROVED** [review]

File: content/en/news/releases/1.10.x/announcing-1.10.1/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (craigbox) [reply content/en/news/releases/1.10.x/announcing-1.10.1/index.md:20]: **that was** making it?

Comment (Monkeyanator) [reply content/en/news/releases/1.10.x/announcing-1.10.1/index.md]: Thanks, great catch

[the remark this answers is not in the document]
Comment (Monkeyanator) [reply content/en/news/releases/1.10.x/announcing-1.10.1/index.md]: Done

File: content/en/news/releases/1.10.x/announcing-1.10.1/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - /news/announcing-1.10.1
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.10.0 and Istio 1.10.1.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue causing the `Host` header to not be modifiable for specific destinations in `VirtualService` ([Issue #33226](https://github.com/istio/istio/issues/33226))
```
[the remark this answers is not in the document]
Comment (Monkeyanator) [reply content/en/news/releases/1.10.x/announcing-1.10.1/index.md]: Done

File: content/en/news/releases/1.10.x/announcing-1.10.1/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Announcing Istio 1.10.1
+linktitle: 1.10.1
+subtitle: Patch Release
+description: Istio 1.10.1 patch release.
+publishdate: 2021-06-08
```
Comment (ericvn) [inline content/en/news/releases/1.10.x/announcing-1.10.1/index.md:6]: Based on earlier comments, I think this needs to be updated to June 9th.

Comment (Monkeyanator) [reply content/en/news/releases/1.10.x/announcing-1.10.1/index.md]: Done

**Review by ericvn — APPROVED** [review]

**Pull request merged**